src/redux/message/message.js

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+
+const GET_MESSAGE_API = 'messageReducer/GET_MESSAGE_API';
+
+const initialState = [];
+
+const getMessage = (payload) => ({
+  type: GET_MESSAGE_API,
+  payload,
+});
+
+const getMessageAPI = () => async (dispatch) => {
+  const response = await axios.get('http://localhost:4000/v1/messages');
+  const res = await response.data;
+  dispatch(getMessage(res));
+};
+
+// const getMessageAPI = () => async (dispatch) => {
+//   fetch('http://localhost:4000/v1/messages')
+// .then((res) => res.json())
+// .then((data) => {
+//   console.log(data);
+//   dispatch(getMessage(data));
+// });
+// };
```

Comment:
- [OPTIONAL] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your `code base` is always clean and less cluttered, always remember to remove commented-codes.